### PR TITLE
feat(components): Add some tooltip options to existing components

### PR DESCRIPTION
**Pull request by JayBeeDe**
## Title of the Pull Requests

Add some tooltip options to existing components

### Description of the Change

Add some tooltip options to following components:
- oui-switch
- oui-calendar
- oui-timepicker
- oui-inline-adder
- oui-numeric

### Benefits

Add a tooltip action to components that have button or where there is something to click

### Possible Drawbacks

Might be overkill in some cases, to be used with moderation.

### Applicable Issues

MANAGER-13561

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report yarn-build#2878.0 ✘  
* build & test
  * build ✔ 
  * test ✘

File: packages/components/switch/src/js/switch.controller.js
Comment (anooparveti) [inline packages/components/switch/src/js/switch.controller.js:1]: **nitpick:** We cannot depend on the `onChange`. What if user updates the value in their controller?
Maybe we can just accept the tooltip and dev updates the tooltip as and when the user interacts with it.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/components/switch/src/js/switch.controller.js
Comment (JayBeeDe) [reply packages/components/switch/src/js/switch.controller.js:1]: @anooparveti done

File: packages/components/timepicker/src/js/timepicker.component.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ export default {
     onChange: '&',
     onClose: '&',
     onOpen: '&',
+
+    tooltip: '@?',
```
Comment (dectotam) [inline packages/components/timepicker/src/js/timepicker.component.js:27]:
```suggestion
    tooltip: '@?',
```

**Review by dectotam — DISMISSED** [review]

**Review by dectotam — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by benjaminlepine — APPROVED** [review]

**Pull request merged**